cocos/animation/marionette/animation-graph-animation-clip-binding.ts

```diff
@@ -267,6 +293,27 @@ function createRuntimeBindingAG (track: TrackBinding, bindContext: AnimationClip
     return undefined;
 }
 
+class AuxiliaryCurveEvaluation {
+    constructor (private _binding: AuxiliaryCurveBinding, private _curve: RealCurve) {
```

Comment:
default value is `undefined`?

Reply:
Did you mean L293? It's an unrelated code line.

Reply:
Oh, I mean in Line 297, what happen if we call `new AuxiliarayCurveEvaluation()` and would `_binding` and `_curve` will be `undefined`?